specification/web/resource-manager/Microsoft.DomainRegistration/stable/2021-02-01/Domains.json

```diff
@@ -708,6 +708,56 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DomainRegistration/domains/{domainName}/transferOut": {
```

Comment:
This is interpreted by ARM as a PUT on a collection which is not allowed. 

This needs to be either modelled as a POST or needs to be a REST resource PUT for example ... transfertouts/{transfername}